core/install_scss.js

```diff
@@ -0,0 +1,10 @@
+var Download = require('download');
```

Comment:
It doesn't seem to be working on my system. Have you added something to your `package.json` file that didn't get pushed up?